test/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <bringauto/logging/ConsoleSink.hpp>
+#include <bringauto/logging/Logger.hpp>
+
+#include <gtest/gtest.h>
+
+
+using namespace bringauto::logging;
+
+int main(int argc, char **argv) {
+	Logger::addSink<ConsoleSink>();
+	Logger::LoggerSettings params {"FleetHttpClientTests",
+								   Logger::Verbosity::Debug};
```

Comment:
do not align continuation with multiple tabs pls.
It applies everywhere - constructor, initialization, ...

Reply:
fixed formatting